02-express-tutorial/9-middleware-use.js

```diff
@@ -0,0 +1,31 @@
+const express = require ('express')
+const app = express()
+const logger = require('./logger')
+
+
+//req => middleware => res
+
+//applies to relevant paths api/home/about/products
+//app.use('/api', logger)
+// applies to all paths 
+app.use(logger)
+
```

Comment:
app.use(logger) is applying the middleware automatically, so you don't need to pass it in to each route.  
I.E.: app.get('/',logger, (req,res)
Can be: app.get('/', (req,res)